### PR TITLE
Add status property for Test #415

### PR DESCRIPTION
**Pull request by draganakovacevic**

### CODE REVIEW

File: web/src/main/java/eu/execom/testgenerator/web/dto/DashboardTestDto.java [hunk 1]
```diff
@@ -32,6 +33,8 @@ public DashboardTestDto(Test test) {
     this.dateTaken = test.getTimeTaken().toLocalDate();
     this.awardedPoints = test.getAchievedPoints();
     this.maxPoints = test.getMaximumPoints();
+    String currentStatus = test.getStatus().toString();
+    this.status = currentStatus.substring(0, 1) + currentStatus.substring(1).replace("_", " ").toLowerCase();
```
Comment (stanislavgrujic) [inline web/src/main/java/eu/execom/testgenerator/web/dto/DashboardTestDto.java:37]: Why do you need this transformation? Which value do you send to the frontend?

File: web/src/main/java/eu/execom/testgenerator/web/controller/ReviewController.java [hunk 2]
```diff
@@ -86,7 +87,7 @@ public ResponseEntity applyReview(@RequestBody SubmitReviewRequest submitReviewR
 
   private void checkIfTestIsAvailableForReviewer(User user, Test test) {
 
-    if (!test.isCompleted()) {
+    if (!test.getStatus().equals("FINISHED")) {
```
Comment (stanislavgrujic) [inline web/src/main/java/eu/execom/testgenerator/web/controller/ReviewController.java:90]: Isn't the status field of type `TestStatus`? How come you compare it with string value?

File: web/src/main/frontend/src/base/stores/AuthStore.js [hunk 3]
```diff
@@ -1,9 +1,10 @@
 import {observable, action} from 'mobx';
 import authApi from '../rest/authApi';
-import {AUTH_TOKEN_KEY} from '../../components/utility/httpUtil';
+import {AUTH_TOKEN_KEY, USER_IS_ADMIN} from '../../components/utility/httpUtil';
 
 class AuthStore {
-  @observable isAdmin = true;
+
```
Comment (aberonja) [inline web/src/main/frontend/src/base/stores/AuthStore.js:6]: Same comment as in #421

File: web/src/main/frontend/src/base/stores/AuthStore.js [hunk 4]
```diff
@@ -1,9 +1,10 @@
...
+  @observable isAdmin;
```
Comment (aberonja) [inline web/src/main/frontend/src/base/stores/AuthStore.js:8]: Same comment as in #421

File: web/src/main/frontend/src/components/DashboardTableBody.js [hunk 5]
```diff
@@ -32,27 +32,41 @@ class DashboardTableBody extends React.Component {
...
-              />
-              <button
-                className="dashboard__trash_icon"
-                onClick={this.props.confirmPopup.showPopup(this.rowDelete(row))}
-                title="Delete test..."
-              />
-            </td>
-          </React.Fragment>
-        )}
+
```
Comment (aberonja) [inline web/src/main/frontend/src/components/DashboardTableBody.js:35]: Same comment as in #421

File: web/src/main/frontend/src/components/DashboardTableBody.js [hunk 6]
```diff
@@ -32,27 +32,41 @@ class DashboardTableBody extends React.Component {
...
+        <React.Fragment>
+          {this.props.isAdmin === 'true' ?
```
Comment (aberonja) [inline web/src/main/frontend/src/components/DashboardTableBody.js:37]: Same comment as in #421

File: core/src/test/java/eu/execom/testgenerator/core/service/TestServiceTest.java [hunk 7]
```diff
@@ -317,7 +318,7 @@ public void shouldFindNotCompletedTests() {
     when(testRepository.findByCompletedFalse()).thenReturn(Arrays.asList(test1(), test2(), test3()));
     List<Test> notCompletedTests = testService.findAllNotCompleted();
 
-    notCompletedTests.forEach(test -> assertFalse(test.isCompleted()));
+    notCompletedTests.forEach(test -> assertFalse(!test.getStatus().equals("FINISHED")));
```
Comment (stanislavgrujic) [inline core/src/test/java/eu/execom/testgenerator/core/service/TestServiceTest.java:321]: This will never be true, since TestStatus is not of String type.

**Pull request merged**